Applications/GesturePod/onComputer/README.md

```diff
@@ -0,0 +1,46 @@
+GesturePod *(Gesture Recognition Pipeline)*
+=========================
+
+## Table of Contents
+
+- [About](#about)
+- [Quick Start](#quick-start)
+- [Docs](#docs)
+- [Also](#also)
+
+## About
+
+GesturePod is a plug-and-play, gesture recognition device. 
+
+As the GesturePod involves hardware, this implements GesturePod's gesture recognition pipeline on your computer system. We have stored sensor data collected from the GesturePod. This pre-stored sensor data is used for gesture recognition. You can now play around with GesturePod on your computer!
```

Comment:
Could you remove the '!' marks.

Also, change this to the following.

> This repository isolates the gesture recognition pipeline that is part of the `Arduino MKR1000` implementation. All additional modules, for instance, the Bluetooth communication module, that interfaces with the IMU, and so forth are removed. Data is read from a provided text file and prediction is performed. This repository is intended to be used as a playground to understand the gesture recognition pipeline as well as a bare-minimum implementation of gesture recognition that can be used  as a basis for porting GesturePod to devices other than `MKR1000`.

Reply:
Updated with minor modifications.